tests/test_project.py

```diff
@@ -1052,71 +1052,71 @@ def call_subcmd(self, subcmd):
                 with switch_to_directory(self.project.root_directory()):
                     return subprocess.check_output(_cmd.split(), stderr=subprocess.DEVNULL)
         except subprocess.CalledProcessError as error:
-            print(error, file=sys.stderr)
-            print(error.output, file=sys.stderr)
+            print(error, sys.stderr)
```

Comment:
I would prefer to revert these two lines to include `file=`.
```suggestion
            print(error, file=sys.stderr)
```